gulp/tasks/gen-service-worker.js

```diff
@@ -75,9 +75,9 @@ function generateServiceWorker(es6) {
     });
 
     const options = {
-      navigateFallback: '/',
+      navigateFallback: '/states',
```

Comment:
Trying to wrap my head around this one. Based on the documents here https://github.com/GoogleChromeLabs/sw-precache#navigatefallback-string , it seems that `navigateFallback` should be precached in either `staticFileGlobs` or `dynamicUrlToDependencies`. Otherwise looks good.

Reply:
Hmm, you are right - this param didn't have any effect before and it doesn't have any effect now.
Thus navigateFallbackWhitelist is also useless then.

I wonder if I should just delete the whole `navigateFallback` section. WDYT?

Reply:
Seems reasonable to me, although long-term it would probably be good to get the states page precached so we can safely use it as our fallback here.

Reply:
I think the idea was that any path that is not under the `navigateFallbackWhitelist` is supposed to be a panel served by a component. The `/` page will load the app that can show that the panel doesn't exist.

Reply:
Just that now "/" is no longer pre-cached.

Reply:
Removed navigateFallback

Reply:
I guess I missed something, why is `/` no longer pre-cached?

Reply:
It is runtime cached with cache-first policy.
When it was precached changing extra_html_url and other template-injected variables didn't have any effect.